io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,511 @@
+    sources:
+      - type: git
+        url: https://github.com/schrodinger/maeparser
+        tag: v1.3.1
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/maeparser
+      - cp -r * /app/tmp/maeparser
+
+  - name: coordgenlibs
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/schrodinger/coordgenlibs
+        tag: v3.0.2
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/coordgen
+      - cp -r * /app/tmp/coordgen
+
+  - name: RingDecomposerLib
+    sources:
+      - type: git
+        url: https://github.com/rareylab/RingDecomposerLib
+        tag: v1.1.3_rdkit
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/RingDecomposerLib
+      - cp -r * /app/tmp/RingDecomposerLib
+
+  - name: pubchem-align3d
+    sources:
+      - type: git
+        url: https://github.com/ncbi/pubchem-align3d
+        commit: daefab3dd0c90ca56da9d3d5e375fe4d651e6be3
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/pubchem-align3d
+      - cp -r * /app/tmp/pubchem-align3d
+
+  - name: rapidjson
+    sources:
+      - type: git
+        url: https://github.com/Tencent/rapidjson
+        commit: 858451e5b7d1c56cf8f6d58f88cf958351837e53
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/rapidjson-1.1.0
+      - cp -r * /app/tmp/rapidjson-1.1.0
+
+  - name: rdkit
+    sources:
+      - type: archive
+        url: https://github.com/rdkit/rdkit/archive/refs/tags/Release_2023_09_4.tar.gz
+        sha256: abacae431bbc5882b87cc8629b7ddc02757204e854aa45b6157ec2bf45d623ef
+    buildsystem: simple
+    build-commands:
+      - cp -r /app/tmp/RingDecomposerLib External/RingFamilies/
+      - cp -r /app/tmp/pubchem-align3d External/pubchem_shape/
+      - cp -r /app/tmp/maeparser External/CoordGen/
+      - cp -r /app/tmp/coordgen External/CoordGen/
+      - cp -r /app/tmp/rapidjson-1.1.0 External/
+      - mkdir build
+      - cmake -S . -B build -DCMAKE_PREFIX_PATH=/app -DCMAKE_INSTALL_PREFIX=/app -DCatch2_DIR=/app -DPYTHON_EXECUTABLE=/usr/bin/python3 -DPYTHON_INCLUDE_DIR=/usr/include/python3.12 -DRDK_BUILD_CAIRO_SUPPORT=ON -DRDK_INSTALL_INTREE=OFF -DRDK_INSTALL_COMIC_FONTS=OFF -DRDK_BUILD_MAEPARSER_SUPPORT=OFF
+      - cd build && make -j5 && make install
+
+  - name: coot
+    sources:
+      - type: archive
+        url: https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/source/releases/coot-1.1.10.tar.gz
+        sha256: 447caa7bdd6f87b738d20f8db15aa278476c308e22506004ed145e04f85e0413
+      - type: file
+        path: io.github.pemsley.coot.metainfo.xml
+      - type: file
+        path: io.github.pemsley.coot.desktop
+    buildsystem: autotools
+    config-opts:
+      - --with-fftw-prefix=/app
+      - --with-boost=/app
+      - --with-boost-libdir=/app/lib
+      - --with-rdkit-prefix=/app
+      - --with-enhanced-ligand-tools
+      - --with-guile
+      - PYTHON=python3
+      - SHELL=/bin/bash
```

Comment:
```suggestion
```

Reply:
Is python python3?

Reply:
I don't think Freedesktop has preinstalled python2.

Reply:
> you didn't even check?

It is routine for python to be python 2.7 on the computers that I use (not all of them). Do you mean in a flatpak environment? Then no, I didn't - I know very little about flatpak (other than it's a desirable thing) and wouldn't even know how to check.

>  What do you think?

Although unfortunately hfiguiere didn't answer the question, I think we can proceed presuming that python is  python3 and that configuration line is not needed.